src/classes/CountryStateDistrict.cls

```diff
@@ -0,0 +1,115 @@
+public with sharing class CountryStateDistrict {
+    //dynamiclly field Maps
+    Map<string,Map<string,List<string>>> mapOfStatewithCountry=new Map<string,Map<string,List<string>>>();
+
+    Map<string,List<string>> mapOfDistrictsWithState=new Map<string,List<string>>();
+
+    List<string> ListOfDistricts=new List<string>();
+    public  CountryStateDistrict(string ch,integer length) {
```

Comment:
No requirement for this constructor.
for "the length greater than the specific number" functionality, create a separate method with the input of integer. for "start with string", we have a different method as countryJSONGenerator.